modules/src/core/ics04_channel/context.rs

```diff
@@ -279,3 +273,15 @@ pub trait ChannelKeeper {
     /// Should never fail.
     fn increase_channel_counter(&mut self);
 }
+
+pub fn calculate_block_delay(
+    delay_period_time: Duration,
+    max_expected_time_per_block: Duration,
+) -> u64 {
+    let expected_time_per_block = max_expected_time_per_block;
```

Comment:
Is `expected_time_per_block` needed?

Reply:
Removed 👍 https://github.com/informalsystems/ibc-rs/pull/1917/commits/ef95dd2b5350ddd88f192c7932f133ddbf23c817